test/catalyst/gemsCatalystsRegistry.ts

```diff
@@ -20,48 +20,50 @@ describe('GemsCatalystsRegistry', function () {
     );
   });
 
-  it('burnCatalyst should burn 2 common catalysts from catalystOwner account', async function () {
+  it('burnCatalyst should burn 2 common catalysts from catalystMinter account', async function () {
```

Comment:
I think this is a fine test, but I think it might be good to also have a test not involving the catalystMinter.
This account is special and we want to be sure burning work for any account